### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by breyana**

### CODE REVIEW

File: pages/projects.html [hunk 1]
```diff
@@ -30,11 +30,26 @@
...
+      </p>
+      <div>
+        <img src="../images/CookieClickScreenshot.png" width="400px">
+      </div>
+      <h1> To Do List </h1>
+      <a href="https://github.com/breyana/huge-manatee-to-do-list">Github</a></li>
+      <p>
+        This was my first experience using React.
+        The fun part was allowing the user to click the task's text to edit the task
+        and have it saved on the backend.
```
Comment (Moniarchy) [inline pages/projects.html:48]: Why is this text on a newline?

**Review by Moniarchy — APPROVED** [review]
Breyana, this looks great. The only thing I don't see of your specs is twitter integration. Other than this, your project is 100 percent done.

**Review by Moniarchy — COMMENTED** [review]
See above

File: pages/projects.html (hunk 1 above)
Comment (breyana) [reply pages/projects.html:48]: Trying to not run the lines too long past the 50 character mark so it's possibly easier to read

**Comment by breyana** [comment]
@Moniarchy for the Twitter Integration, check the resources page. I don't have a Twitter that I'd like to connect professionally

**Pull request merged**